### ISSUE TITLE
Trackpad only working 1 hour then stopping

### ISSUE THREAD

**Issue by robertojguerra** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#136]
Hello... I don't know whether I should post my problem here, in VoodooPS2, VoodooRMI or VoodooI2C, since this project uses all 3 kexts.
I use a Yoga 260... is everyone else on T460 not suffering this problem? After 1 hour of use, the trackpad stops working and I can only continue with a USB mouse.

**Comment by simprecicchiani** [comment]
Does this happen with older releases?

**Comment by robertojguerra** [comment]
Oh sorry I forgot to add: I am using Monterey and I haven’t tried BigSur.

**Comment by robertojguerra** [comment]
I found from the kreizlie GitHub project, that disabling the red trackpoint in BIOS, prevents the trackpad from stopping.

**Comment by cyberfunk** [comment]
I have a new issue where the trackpad will stop responding after an undetermined amount of time. My computer does not freeze it just seems like the trackpad disconnects from the system and will not resume functioning until I reboot. I can plug in a USB mouse and continue working when it happens but the trackpad just stops working. Does anyone have a kext or other fix for this behavior?

**Comment by robertojguerra** [comment]
Did you try disabling the red “nipple” in the bios?

**Comment by cyberfunk** [comment]
Yes I'm the one who came up with that solution to stop the computer from freezing. This is different though. The computer does not freeze but the trackpad simply stops functioning altogether.
